app/views/users/profile.html.erb

```diff
@@ -239,6 +245,18 @@
     alert("Your access token is: <%= @user.user.token %>");
   });
 
+$('#link_social_media').click(function(){
+   var sign = prompt("Enter Your Social Media User name like twitter:jywarren or instagram:unterbahn");
+    $.ajax({
+       method: "POST",
+       url: "http://localhost:3000/users/link/<%=@user.uid %> ",
```

Comment:
Indeed - as pointed out, here you could remove the `http://localhost:3000` -- good catch!